package.json

```
@@ -19,7 +19,9 @@
     "lint": "eslint .",
     "start": "node src/shared/cli/index.js",
     "qa": "pta ./src/qa/*.qa.js -r log | node src/qa/utils/qaReporter.js",
-    "test": "FILES_MODIFIED=$(git diff --name-only HEAD) jest",
+    "test": "npm run test:jest && npm run test:tape",
+    "test:tape": "tape tests/**/*-test.js | tap-spec",
```

```suggestion
    "test:tape": "tape -r esm tests/**/*-test.js | tap-spec",
```

This avoids having to import `esm` in every test file

---

I'm not sure what side effects that will have, and I don't believe it's necessary at this time.